post-til-virksomhet/src/main/java/no/difi/meldingsutveksling/ptv/CorrespondenceAgencyMessageFactory.java

```diff
@@ -123,6 +95,12 @@ public static InsertCorrespondenceV2 create(CorrespondenceAgencyConfiguration po
         externalContentV2.setAttachments(objectFactory.createExternalContentV2Attachments(attachmentsV2));
         correspondence.setContent(objectFactory.createMyInsertCorrespondenceV2Content(externalContentV2));
 
+        List<Notification2009> notificationList = createNotifications(postConfig, edu);
+
+        NotificationBEList notifications = new NotificationBEList();
+        notificationList.forEach(n -> notifications.getNotification().add(n));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1612)